docs/modules/ROOT/pages/explanation/organizations-and-rbac.adoc

```diff
@@ -20,3 +26,21 @@ Membership of organizations and organizations itself are currently being maintai
 
 It's planned to enable self-management of organizations and membership in the https://roadmap.appuio.cloud[upcoming Webportal].
 Please xref:contact[contact] us to manage organization membership for you for the time being.
+
+== Transfer Projects
+
+There is the option to move a project from one organization to another.
+This can be done by setting the label `appuio.io/organization` on the `Namespace` of the project to the name of the target organization.
+This needs to be done by someone in the target organization with the permission to edit the `Namespace`.
+
+.As an example: 
+Let `foo` be a namespace that belongs to the organization `purple-fox`.
+Let user `john` be part of the organization `carbon-fund`.
+To transfer the namespace `foo` to organization `carbon-fund`, the user `john` needs to have the permission to edit the namespace `foo`.
+With the default RBAC rules, this means he needs to have the role `namespace-owner`.
+With that the user `john` can move the project `foo` to the organization `carbon-fund`, by setting the label `appuio.io/organization` on the namespace `foo` to `carbon-fund`.
+This can't be done by a user in organization `purple-fox`.
+
+IMPORTANT: This will transfer to project to the new organization and update, among other things, billing.
+It will however not update any RBAC rules.
+You will need to manually change these.
```

Comment:
Also here I'd prefer explicit enclosure
```suggestion
[IMPORTANT]
====
This will transfer to project to the new organization and affect, among other things, billing.
It will however not update any RBAC rules.
You will need to manually change these.
====
```